### ISSUE TITLE
Support for component TemplateUrl and StylesUrl

### ISSUE THREAD

**Issue by raparlasiva** [driftyco/ionic2-module-template#8]
it would be nice if the repo can add support to have templateUrl and stylesUrl as it would nice to have separate files for html and css/scss. Also like to see support for relative paths when adding component specific styles and template.

**Comment by stefanhuber** [comment]
Absolutely, this is not working! templates referenced by urls from NgModules are not found

**Comment by fedecastelau** [comment]
I have the same problem. Did you find any solution? thank

**Comment by gciluffo** [comment]
Also looking for a solution for this. Anybody find any workaround for this for now?

**Comment by fedecastelau** [comment]
@gciluffo Yes, im using [this repo](https://github.com/jvandemo/generator-angular2-library) !

**Comment by mlynch** [comment]
I'm not sure why this isn't working. Anyone have any ideas?

ngc doesn't seem to support this out of the box, and that repo is using a script. Given that, I'm hesitant to add custom code here as this should be supported in Angular to fit the spirit of this repo.

**Comment by frystpingvin** [comment]
Any update on this? Going to modularize an Ionic app, but this might be a dealbreaker.

**Comment by 378740021** [comment]
we are going to modularize our Ionic app. We plan to assign modules to different teams to develop. Any solution to templateUrl now? @mlynch
